src/app/containers/Preferences/Visualizer/VisualizerOptions.js

```diff
@@ -39,6 +39,10 @@ const VisualizerOptions = ({ state, actions }) => {
                     </Tooltip>
                     <ToggleSwitch checked={objects.cutPath.visible} onChange={() => visualizerActions.handleCutPathToggle()} size="md" />
                     <ToggleSwitch checked={objects.cutPath.visibleLite} onChange={() => visualizerActions.handleCutPathToggle(true)} size="md" />
+                    <Tooltip content="Show warning when current workspace 0 will cause the machine to cut outside of soft limits" location="default">
```

Comment:
I'd consider disabling this as a selectable option if $20 (soft limits) isn't enabled on the firmware side